### DISCUSSION TITLE
How should the port and address be set in Bungee mode?

### DISCUSSION THREAD

**Discussion by FZDNSZYT** [plan-player-analytics/Plan#2725] [Q&A] [answered]
How should this port and address be set in Bungee mode? If I don’t set it, it will report an error. If I set it to waterfall, it will also report an error.
![image](https://user-images.githubusercontent.com/86947716/202375188-c3553798-0ff9-4b40-8d72-dcefd99f5b7d.png)

**Comment by AuroraLS3** [comment, marked as answer]
What error?

**Reply by FZDNSZYT** [reply]
![image](https://user-images.githubusercontent.com/86947716/202905430-099be1c1-c971-4b7a-b65f-e6e55a2c8006.png)
he pauses at this step，Even in the past day. Can this process be done manually?

**Reply by FZDNSZYT** [reply]
Has been downloading, the version is 5.5 build 1984

**Reply by AuroraLS3** [reply]
Put these jars in /plugins/Plan/libraries folder
[libraries.zip](https://github.com/plan-player-analytics/Plan/files/10049928/libraries.zip)

**Reply by FZDNSZYT** [reply]
OK,
![image](https://user-images.githubusercontent.com/86947716/202909719-a06b081d-7667-4915-a9f2-3b803967baef.png)

**Reply by FZDNSZYT** [reply]
![image](https://user-images.githubusercontent.com/86947716/202909751-9885ce71-64b9-403b-b78f-69814a8874fc.png)
It's the file that can't be downloaded.

**Reply by FZDNSZYT** [reply]
Is his library down

**Reply by AuroraLS3** [reply]
Probably limitation in your region
